src/lib/select/_select-theme.scss

```diff
@@ -34,7 +34,7 @@
     }
   }
 
-  .md-select-content {
+  .md-select-panel {
```

Comment:
@crisbeto Given that the color should come in immediately after the animation (rather than animating in), it might be easier to do this outside of the animation definition. Have you tried adding something to the existing "done" hook for the animation?  There is an _onPanelDone method that is called when the animation finishes. Try toggling a style or class binding at that point instead.